docs/source/performance/cache-backends.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Configuring cache backends
+description: How to configure Apollo Server's cache
+---
+
+> ⚠️ New in Apollo Server 3.9: we **highly recommend** that all users pass `cache: "bounded"` or configure their cache in a manner which isn't unbounded (which is current default behavior) to protect your server from attacks which exhaust available memory causing a DOS. Continue reading for more information on configuring Apollo Server's cache.
+
+Apollo Server has several features that take advantage of a cache backend (e.g., [automatic persisted queries](./performance/apq), the [response cache plugin](./performance/caching#caching-with-responsecacheplugin-advanced), and [`RESTDataSource`](./data/data-sources#restdatasource-reference)). By default, Apollo Server uses a cache in memory, but you can configure it to use a different backend such as Redis or Memcached.
+
+You can specify a cache backend by passing a `cache` option to the `ApolloServer` constructor. Your specified cache backend _must_ implement the [`KeyValueCache`](https://github.com/apollographql/apollo-utils/tree/main/packages/keyValueCache#keyvaluecache-interface) interface from the `@apollo/utils.keyvaluecache` package.
```

Comment:
`_must_ implement` seems kinda aggressive; maybe just `implements` or `should implement` or even just not italics? Unless this was a suggestion from Docs

Reply:
Haha, that was a suggestion from me for added emphasis, I don't feel strongly one way or another though!

Reply:
Will remove italics. "implements" sounds a little weird, "should implement" is a lot less correct than "must implement".